.gitignore

```diff
@@ -6,9 +6,30 @@
 pubspec.lock
 .example/lib/application_constants.dart
 
+# IntelliJ related
+*.iml
+*.ipr
+*.iws
+.idea/
+
+# Android related
+gradle.properties
```

Comment:
I think, gradle.properties should not be ignored. @phillwiggins Do you have any comments? Let me know if I need to fix this.

Reply:
Never mind, this is fixed now.

Reply:
Sorry for the delay, I've been away. Yes, you've done the correct thing by not adding gradle.properties to gitignote. Thanks